properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
+name: Linearly orderable
+aliases:
+  - Linearly ordered topological space
+  - LOTS
+refs:
+  - wikipedia: Order_topology
+    name: Order topology
```

Comment:
Move the wikipedia reference to the end of the list, as it's always preferable to have published books or peer reviewed journal articles first.  Also for consistency with the rest of pi-base, we usually add "on Wikipedia" to the end of the name field (see https://topology.pi-base.org/properties/P000090/references for a random example).  Makes it easier for the reader to know it's from Wikipedia.